Source/UnitTests/Core/IOS/ES/TestBinaryData.cpp

```diff
@@ -0,0 +1,224 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "TestBinaryData.h"
+
+const u8 soup01_tmd[] =
```

Comment:
These can just be exposed as `std::array` instances can they not? It would get rid of the need for an extra size variable.